### PR TITLE
feat(star_aln): more commands

### PR DESCRIPTION
**Pull request by jemten**
### This PR fixes:
- more star commands needed for arriba

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Program/Star.pm [hunk 1]
```diff
@@ -98,7 +118,13 @@ sub star_aln {
             store       => \$align_sjdb_overhang_min,
             strict_type => 1,
         },
+        align_sj_stitch_mismatch_nmax => {
+            allow       => qr/\A (?: -?\d+\s ){3} -?\d \z /xms,
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Star.pm]: Would be nice with a inline comment on the regexp i.e. what it is supposed to match

File: lib/MIP/Program/Star.pm [hunk 2]
```diff
@@ -158,6 +219,16 @@ sub star_aln {
             store       => \$out_sam_type,
             strict_type => 1,
         },
+        out_sam_unmapped => {
+            allow       => [ qw{ None Within }, q{Within KeepPairs} ],
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Star.pm]: Just dubbelchecking that q{Within KeepPairs} are intended to be one word here

File: lib/MIP/Program/Star.pm [hunk 3]
```diff
@@ -35,42 +35,62 @@ sub star_aln {
...
+##           : $infile_paths_ref              => Fastq file path(s)
+##           : $limit_bam_sort_ram            => Memory available for sorting the output bam
+##           : $outfile_name_prefix           => Prefix of the output files (remember to end with a ".")
+##           : $out_bam_compression           => Compression level of BAM file
+##           : $out_filter_mismatch_nmax      => Max number of missmatches allowed in alignment
+##           : $out_filter_multimap_nmax      => Max number of loci a read is allowed to map to
+##           : $out_sam_strand_field          => Cufflinks-like strand field flag
+##           : $out_sam_type                  => Format of the output aligned reads
+##           : $out_sam_unmapped              => Write unmapped reads to main BAM file
+##           : $out_std                       => Which output will be directed to stdout
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Star.pm]: a bit unclear what this really is. Maybe "outype_to_stdout"

File: lib/MIP/Program/Star.pm (hunk 2 above)
Comment (jemten) [reply lib/MIP/Program/Star.pm]: yes. The two words needs to be paired.

File: lib/MIP/Program/Star.pm (hunk 3 above)
Comment (jemten) [reply lib/MIP/Program/Star.pm]: how about "stdout_data_type"?

**Pull request merged**